doc/04-Upgrading.md

```diff
@@ -3,6 +3,49 @@
 Specific version upgrades are described below. Please note that version upgrades are incremental.
 If you are upgrading across multiple versions, make sure to follow the steps for each of them.
 
+## Upgrading to Icinga DB v1.1.2
+
+Please apply the `1.1.2.sql` upgrade script to your database.
+For package installations, you can find this file at `/usr/share/icingadb/schema/mysql/upgrades/` or
+`/usr/share/icingadb/schema/pgsql/upgrades/`, depending on your database type.
+
+As the daemon checks the schema version, the recommended way to perform the upgrade is to stop the daemon, apply the
+schema upgrade and then start the new daemon version. If you want to minimize downtime as much as possible, it is safe
+to apply this schema upgrade while the Icinga DB v1.1.1 daemon is still running and then restart the daemon with the
+new version. Please keep in mind that depending on the distribution, your package manager may automatically attempt to
+restart the daemon when upgrading the package.
+
+### Upgrading the state_history Table (optional)
+
+Icinga DB 1.1.1 already handles hosts/services' `max_check_attempts` above 255.
+But actual soft state changes with check attempt above 255 in your history will crash Icinga DB,
+even after upgrading to v1.1.2. The `check_attempt` column of the `state_history` table is too small
+to fit larger values. ([#655](https://github.com/Icinga/icingadb/issues/655))
+
+Databases created with the v1.1.2 schema are not affected, but upgraded ones are.
+The schema upgrade required to fix this isn't included in `1.1.2.sql` as it re-writes the whole `state_history` table.
+This can take a lot of time depending on the history size and the performance of the database.
+During this time that table will be locked exclusively and can't be accessed otherwise.
+This means that the existing history can't be viewed in Icinga Web and new history entries will be buffered in Redis.
+
+That optional table re-write is given below and should be applied during the next longer maintenance window.
+Do not interrupt it! At best use tmux/screen not to loose your SSH session.
+Until this upgrade you'll have to lower `max_check_attempts`. Optionally increase the respective `retry_interval`
+not to change the timespan from an outage to a hard state (`max_check_attempts x retry_interval`).
```

Comment:
`x` as a multiplication sign inside a code block looks strange to me. IMHO all of these would fit better:

- `max_check_attempts * retry_interval`
- `max_check_attempts` × `retry_interval` (unicode multiplication sign)
- `max_check_attempts` times `retry_interval`